packages/sdk-js/test/features.test.ts

```diff
@@ -200,11 +200,14 @@ describe("features", () => {
 
     const growthbook = new GrowthBook(context);
 
-    growthbook.setAttributes({ foo: 2, baz: 3 });
+    const dateTime = new Date("2022-11-01").toISOString();
+
+    growthbook.setAttributes({ foo: 2, baz: 3, current_datetime: dateTime });
 
     expect(context.attributes).toEqual({
       foo: 2,
       baz: 3,
+      current_datetime: dateTime,
```

Comment:
I don't love this. Is there an equality method I can test here that will ignore the `current_datetime` property?

Reply:
Check out `expect.objectContaining({key1: val1, key2: val2})` for partial matches